### PR TITLE
Modernize GeoBlacklight tutorial

### PR DESCRIPTION
**Pull request by mejackreed**
This pull request aims to update the tutorial to work with the current version of GeoBlacklight.

Requires: https://github.com/geoblacklight/geoblacklight/pull/998

### CODE REVIEW

File: _posts/2015-02-09-setting-up-your-environment.markdown [hunk 1]
```diff
@@ -18,13 +18,16 @@ GeoBlacklight has similar prerequisites to [Blacklight][bldependencies]. It dive
...
-  - [Rails > 5][installrails]
+  - [Ruby > 2.6.6][installruby]
+  - [Rails > 6.0][installrails]
   - [Git][installgit]
   - [Java > 1.8][installjava] (Download JDK for local Solr server)
-  - [Node.js > 10.15 LTS][installnode]
+  - [Node.js > 14.15 LTS][installnode]
   - [Yarn > 1.13][installyarn]
 
+It is reccomended to install the latest versions of Ruby, Rails, and Node.js. We strive to keep GeoBlacklight updated with these versions. A great, almost always up-to-date, tutorial on getting a Ruby on Rails development environment is available here: [https://gorails.com/setup](https://gorails.com/setup). If you are not following this tutorial as part of an in person workshop, you can skip to the next section once you have these dependencies installed.
```
Comment (karenmajewicz) [inline _posts/2015-02-09-setting-up-your-environment.markdown:28]: spelling typo: "recommended"

File: _posts/2015-02-09-geoblacklight-overview.markdown [hunk 2]
```diff
@@ -32,17 +32,19 @@ That's ok. GIS data indexed into GeoBlacklight becomes progressively more useful
 
 [The GeoBlacklight Schema, Version 1.0](https://github.com/geoblacklight/geoblacklight/blob/master/schema/geoblacklight-schema.md) uses a field `dct_references_s` to define external services and references. See the [references.md](https://github.com/geoblacklight/geoblacklight/blob/master/schema/references.md) portion of the repository for a list of possible key:value pairs and instructions on how to apply them.
```
Comment (karenmajewicz) [inline _posts/2015-02-09-geoblacklight-overview.markdown:33]: Links are to the Markdown files, which are possibly a bit out of date. We could update this later though.

File: _posts/2015-02-09-setting-up-your-environment.markdown (hunk 1 above)
Comment (mejackreed) [reply _posts/2015-02-09-setting-up-your-environment.markdown]: Nice catch! Fixed

File: _posts/2015-02-09-create-your-application.markdown [hunk 3]
```diff
@@ -118,7 +116,7 @@ Now that we have started our Rails application, we need to install GeoBlacklight
      {: .flash-notice}
 
      ```sh
-     $ rails new your_app_name -m https://raw.githubusercontent.com/geoblacklight/geoblacklight/master/template.rb
+     $ DISABLE_SPRING=1 rails new your_app_name -m https://raw.githubusercontent.com/geoblacklight/geoblacklight/main/template.rb
```
Comment (karenmajewicz) [inline _posts/2015-02-09-create-your-application.markdown:119]: We might want to remind users to cd into the new directory before running the next rake task?

**Review by karenmajewicz — COMMENTED** [review]
One thing that would be nice would be a "Date Updated" to let people know that these tutorials have been worked on recently. Is that a Jekyll meta field?

File: _posts/2015-02-09-create-your-application.markdown (hunk 3 above)
Comment (mejackreed) [reply _posts/2015-02-09-create-your-application.markdown:119]: Good point! I've pushed up that change.

File: _posts/2015-02-09-geoblacklight-overview.markdown (hunk 2 above)
Comment (mejackreed) [reply _posts/2015-02-09-geoblacklight-overview.markdown]: I've linked the wiki pages now. Thanks for the suggestion.

**Pull request merged**